src/renderer/helpers/accentColors.ts

```diff
@@ -39,3 +40,7 @@ export const getTypeAccentColors = (inputType: Type): readonly [string, ...strin
     }
     return colors.length > 0 ? (colors as [string, ...string[]]) : defaultColorList;
 };
+
+export const getCategoryAccentColor = (categories: Category[], category: string) => {
```

Comment:
We don't mutate the array.

```suggestion
export const getCategoryAccentColor = (categories: readonly Category[], category: string) => {
```